Disable the warning that appears when editing a file being viewed with
LESSOPEN.

--- command.c	2026-09-08 20:42:35.000000000 -0500
+++ command.c	2026-10-28 16:07:46.154033593 -0500
@@ -2159,10 +2159,6 @@
 					error(LM(Cannot_edit_standard_input), NULL_PARG);
 					break;
 				}
-				if (!no_edit_warn && get_altfilename(curr_ifile) != NULL)
-				{
-					error(LM(This_file_was_viewed_via_LESSOPEN), NULL_PARG);
-				}
 				start_mca(A_SHELL, "!", ml_shell, 0);
 				/*
 				 * Expand the editor prototype string
